GitUI/UserEnvironmentInformation.cs

```diff
@@ -27,12 +33,53 @@ public static string GetInformation()
 
             StringBuilder sb = new StringBuilder();
 
-            sb.AppendFormat("- Git Extensions {0}{1}", AppSettings.ProductVersion, Environment.NewLine);
-            sb.AppendFormat("- {0} {1}{2}", ThisAssembly.Git.Sha, ThisAssembly.Git.IsDirty ? " (Dirty)" : "", Environment.NewLine);
-            sb.AppendFormat("- Git {0}{1}", gitVer, Environment.NewLine);
-            sb.AppendFormat("- {0}{1}", Environment.OSVersion, Environment.NewLine);
-            sb.AppendFormat("- {0}", RuntimeInformation.FrameworkDescription);
+            sb.Append($"- Git Extensions {AppSettings.ProductVersion}{Environment.NewLine}");
+            sb.Append($"- {_sha} {(_dirty ? " (Dirty)" : "")}{Environment.NewLine}");
+            sb.Append($"- Git {gitVer}{Environment.NewLine}");
+            sb.Append($"- {Environment.OSVersion}{Environment.NewLine}");
+            sb.Append($"- {RuntimeInformation.FrameworkDescription}");
             return sb.ToString();
         }
+
+        public static void Initialise(string sha, bool isDirty)
+        {
+            if (_alreadySet)
+            {
+                return;
+            }
+
+            _alreadySet = true;
+            _sha = sha;
+            _dirty = isDirty;
+        }
+
+        public static bool Dirty
```

Comment:
The Dirty, Sha properties can be removed

Reply:
Durp. Thanks. I spent too long in front of the computer. Should have seen that.